docs/src/components/Image/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import useThemeContext from '@theme/hooks/useThemeContext';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+const Image = ({path}) => {
```

Comment:
This name seems a bit generic. `ContrastImage`, `ThemeImage`, `DarkModeImage`

Either that or we keep it as `Image` and make the props explicit. I.e. `{dark,light}ModePath=`

What do you think? :)

Reply:
I think it would prefer the second option since currently it also forces the use of `png`

Reply:
I did not have a good name/idea for this so I "made it work" and hoped for such a comment.

So I like this:

```js
<Image lightModePath="..." darkModePath="..." />
```

it's a little verbose but maybe that's a good thing. It's also more flexible than the current component. I'd argue I should keep both props required so that it's always obvious which image is used where?

Reply:
Yes I agree with that!

Reply:
will change!